plugins/modules/vmware_guest_disk.py

```diff
@@ -718,6 +725,16 @@ def sanitize_disk_inputs(self):
                                           " 'disk_mode' value from ['persistent', 'independent_persistent', 'independent_nonpersistent']." % disk_index)
             current_disk['disk_mode'] = temp_disk_mode
 
+            # Sharing mode of Disk
```

Comment:
This function is already way to long, could you move this block out in a function. something like: `get_sharing()`.

Reply:
I refactored disk sharing into the get_sharing method with 855a1b305bc6e39ed5c1942753a01eba05e89236. I'll proceed to work on the test cases.